README.md

```diff
@@ -22,20 +22,20 @@ Hints:
     * [tech-hiring on bitbucket](https://bitbucket.org/tech-hiring);
     * [tech-hiring on gitlab](https://gitlab.com/tech-hiring).
 2. Follow the instructions of README.md (this file);
-3. Deploy your project on a host service (we recommend [Heroku](https://heroku.com));
+3. Deploy your project on a hosting service (we recommend [Heroku](https://heroku.com));
 4. Apply for the position at our [career page](https://olist.gupy.io/) with:
     * Link to the fork on Github (or bitbucket/gitlab);
-    * Link to the project in a the deployed host service.
+    * Link to the deployed project in a hosting service.
 
 ## Specification
 
 You should implement an application for a library to store book and authors data.
 
-**This application must provide a HTTP REST API to attend the requirements.**
+**This application must provide an HTTP REST API to attend the requirements.**
```

Comment:
Confesso que fui atrás disso (https://www.merriam-webster.com/words-at-play/is-it-a-or-an), haha, mas acho que rola `an HTTP REST API`.

Reply:
É estranho mas é por causa do som da palavra. Começa com som de vogal. ahaha